src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -1232,6 +1307,23 @@ bool ITokenCacheInternal.HasTokensNoLocks()
             return Accessor.HasAccessOrRefreshTokens();
         }
 
+        /// <summary>
+        /// Removes obo tokens stored in the cache. Note that the cache is internally and externally partitioned by the oboKey.
+        /// </summary>
+        internal /* internal for test only */ void RemoveOboTokensInternal(string oboPartitionKey, RequestContext requestContext)
```

Comment:
Doesn't seem to be used in tests.

Reply:
the internal method is used but it makes sense to use the public method as well